pkg/fetch/vendir.go

```diff
@@ -327,3 +336,25 @@ func (v *Vendir) configMapBytes(configMapRef vendirconf.DirectoryContentsLocalRe
 
 	return kyaml.Marshal(configMap)
 }
+
+// This function only works on image refs. If in the future we decide to
+// expand this option to other fetch options, we will need to add hostname
+// extraction for those
+func (v *Vendir) shouldSkipTLSVerify(url string) bool {
+	hostname := v.extractImageRefHostname(url)
+	log.Printf("HOSTNAME:%s", hostname)
+	skip := v.skipTLSConfig.ShouldSkipTLSForDomain(hostname)
+	log.Printf("SKIP: %v", skip)
```

Comment:
WDYT of the following for here:

```suggestion
	if skip {
	  log.Printf("Skip TLS verify for hostname: %s", hostname)
	}
```

Or do we want to log this in general?

Reply:
Oh, this is my bad. I left that in when debugging the weirdness around the error imgpkg was printing. I can remove the log all together

Reply:
Updated. Thanks for catching this!